src/main/java/com/stuypulse/robot/commands/ElevatorToHeight.java

```diff
@@ -0,0 +1,39 @@
+package com.stuypulse.robot.commands;
+
+import com.stuypulse.robot.subsystems.IElevator;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ElevatorToHeight extends CommandBase {
+
+    private static final double EPSILON = Units.inchesToMeters(1);
+    
+    private final IElevator elevator;
+    private final double height;
+
+    private boolean instant;
+
+    public ElevatorToHeight(IElevator elevator, double height) {
+        this.elevator = elevator;
+        this.height = height;
+
+        instant = true;
+
+        addRequirements(elevator);
+    }
+
```

Comment:
add `public final ElevatorToHeight untilReady()` as a decorator to make this command not instant